docs/source/extending/parsers.rst

```diff
@@ -75,6 +75,22 @@ To add a new fitting problem definition type, complete the following steps:
      the list, raising an issue if the result is not suitably close to the
      specified value.
 
+   - **Jacobian tests** (optional): If the parser you add has analytic Jacobian
+     information which you wish to use, then in ``test_parsers.py`` add the
+    ``<format_name>`` to the ``JACOBIAN_ENABLED_PARSERS`` global variable. Then, 1 file
+     can be provided in the directory to test that Jacobian evaluation is as expected.
+     This file must be in json format and contain a string of the form::
```

Comment:
This formats really weirdly for me:
![image](https://user-images.githubusercontent.com/13645545/85539524-d73ab380-b60d-11ea-851a-9e64ac82f316.png)

Reply:
Fixed